clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -569,7 +589,9 @@
     "nyc": "^15.1.0",
     "patch-package": "^6.4.7",
     "prettier": "2.8.7",
+    "process": "^0.11.10",
```

Comment:
Do we still need this after webpack has been removed?

Reply:
And similarly for all the other dependencies added in the PR. Please, check if they can be removed. It looks like some of them are a ripple effect of having had webpack.

Reply:
Resolved in latest commit.